website/docs/tutorial/refactoring-legacy-sql.md

```diff
@@ -0,0 +1,249 @@
+3. Choose a refactoring strategy
+4. Implement CTE groupings and cosmetic cleanup
+5. Separate data transformations into standardized layers
+6. Audit the output of dbt models vs legacy SQL
+
+Let's get into it!
+
+> Note: this tutorial is excerpted from the new dbt Learn On-demand Course, "Refactoring from Procedural SQL to dbt" - if you're curious, pick up the [free refactoring course here](https://courses.getdbt.com/courses/refactoring), which includes example and practice refactoring projects.
+
+## Migrate your existing SQL code
+
+<WistiaVideo id="5u67ik9t66" />
+
+Your goal in this initial step is simply to use dbt to run your existing SQL transformation, with as few modifications as possible. This will give you a solid base to work from.
+
+While refactoring you'll be **moving around** a lot of logic, but ideally you won't be **changing** the logic. More changes = more auditing work, so if you come across anything you'd like to fix, try your best to card that up for another task after refactoring! We'll save the bulk of our auditing for the end when we've finalized our legacy-to-dbt model restructuring.
+
+To get going, you'll copy your legacy SQL query into your dbt project, by saving it in a `.sql` file under the `/models` directory of your project. 
+
+<Lightbox src="/img/tutorial/refactoring/legacy-query-model.png" title="Your dbt project's folder structure" />
+
+Once you've copied it over, you'll want to `dbt run` to execute the query and populate the table in your warehouse.
+
+> If this is your first time running dbt, you may want to start with the [Introduction to dbt](/docs/introduction) and the [Getting Started tutorial](/tutorial/setting-up) before diving into refactoring.
+
+This step may sound simple, but if you're porting over an existing set of SQL transformations to a new SQL dialect, you will need to consider how your legacy SQL dialect differs from your new SQL flavor, and you may need to modify your legacy code to get it to run at all.  
+
+This will commonly happen if you're migrating from a [stored procedure workflow on a legacy database](https://getdbt.com/analytics-engineering/case-for-elt-workflow/) into dbt + a cloud data warehouse.
+
+Functions that you were using previously may not exist, or their syntax may shift slightly between SQL dialects. 
+
+If you're not migrating data warehouses at the moment, then you can keep your SQL syntax the same. You have access to the exact same SQL dialect inside of dbt that you have querying directly from your warehouse.
+
+## Create sources from table references
+
+<WistiaVideo id="m1a5p32rny" />
+
+To query from your data warehouse, we recommend creating [sources in dbt](/docs/building-a-dbt-project/using-sources) rather than querying the database table directly.
+
+This allows you to call the same table in multiple places with `{{ src('my_source', 'my_table') }}` rather than `my_database.my_schema.my_table`.
```

Comment:
Curious while I'm here - does this work now? I missed an update 🤯

Reply:
Tested `{{ src() }}` with 0.20.2 and it didn't work - so this just needs to be changed to `{{ source() }}` =)